.github/workflows/ci.yml

```diff
@@ -77,17 +63,18 @@ jobs:
         run: ./scripts/github-ci.sh android
         env:
           OS_NAME: linux
-          CI_IMAGE: ${{ needs.setup-env.outputs.ci_image }}
       - name: Upload APK
         id: upload
         uses: actions/upload-artifact@v4
         with:
-          path: frontends/android/BitBoxApp/app/build/outputs/apk/debug/app-debug.apk
+          path: /github/home/go/src/github.com/BitBoxSwiss/bitbox-wallet-app/frontends/android/BitBoxApp/app/build/outputs/apk/debug/app-debug.apk
```

Comment:
unusual to see absolute paths like this - why does relative not work?

Reply:
It fails with with a "not found" error: https://github.com/bznein/bitbox-wallet-app/actions/runs/15879899679/job/44777165124

I wasn't able to figure out at which path is the step running... I tried adding a debug print on the step before and it looks like it should be able to find it: https://github.com/bznein/bitbox-wallet-app/actions/runs/15921416849/job/44908858374

But it doesn't.. I suppose there is some difference when running on a specifid container.

If it's a big deal, I can try figuring out what's happening anyway!